### ISSUE TITLE
UILabel and view properties not animating inside component

### ISSUE THREAD

**Issue by jewilhel** [alexdrone/Render#48]
I have a card component that renders with animation options:

```swift
        card.render(in: view.bounds.size, options: [
            .animated(duration: 1, options: [.curveEaseInOut]) {
                self.card.center = self.view.center
            }
            ])
```

Most child components of type, subtype UIView will animate as well:

``` swift
         let titlebar = Node<UIView> { (view, layout, size) in
                view.backgroundColor = .green
                layout.width = size.width
                layout.aspectRatio = TitleBarSize.InverseProportion
                layout.position = .absolute
                layout.bottom = 0
                layout.alignContent = .center
        }
```

But child component frame/bounds of type UILabel will not animate, but end up sized correctly at the end of the animation. Does the Render framework not support this?

```swift
            let title = Node<UILabel>() { (view, layout, size) in
                view.text = state?.title
                view.numberOfLines = 1
                view.font = Label.font.withSize(size.width * Label.FontSizeRatio)
                view.textColor = UIColor.white
                view.textAlignment = .center
                view.backgroundColor = .darkGray
                layout.width = size.width
                layout.aspectRatio = TitleBarSize.InverseProportion
           }
```

Also, other properties will not animate such as cornerRadius or in the case above, I'm resizing the font based on the view width, but does not animate the property. Is there a way to do this?

Would like to see more examples of animating component views and their properties.

Thanks in advance! Love the framework so far!

**Comment by alexdrone** [comment]
is it just the first time that is not animated? 
when views are being added or removed from the hierarchy for the first time - they won't be animated

**Comment by jewilhel** [comment]
No, UILabel's never animates, unlike the other derivatives of UIView, that I tested. UIView, UIImageView, and UIButton animate just fine. I'm not sure what's wrong, I've tried all sorts of layout options, but UILabel's frame never animates even though it updates to the new state just fine.

I can change the node type from UILabel to UIView or UIButton on the same component and it will start animating. There is either something wrong with the UILabel node type or perhaps it's width is being controlled by its content somehow. 

So to be clear, I'm expecting the UILabel node to resize when its parent component Card is re-rendered.
